Tella/Scenes/Settings/ViewModel/UwaziServerViewModel.swift

```diff
@@ -314,5 +323,39 @@ class UwaziServerViewModel: ObservableObject {
         self.username = server.username ?? ""
         self.password = server.password ?? ""
     }
+    
+    
+    fileprivate func addPrivateInstanceServer() {
```

Comment:
I think it's better to use one function for both `checkURL()` and  `addPrivateInstanceServer` and if we have token it will be included else it will be nil,  it's a duplicated code

Reply:
yeah you are right. I removed the addPrivateInstanceServer function and create just one flow using checkURL. the flow will be like this:

- user enter URL
- We check the URL to see if we enable public instance or go straight to login
    - if URL is public: show both options -> user login -> select lang -> create server
    - if URL is private: user login -> select lang -> checkURL -> create server